src/adapters/postgres/cohort-adapter.ts

```diff
@@ -461,7 +461,10 @@ export class PostgresCohortService {
       let { limit, page, filters } = cohortSearchDto;
 
       let offset = 0;
-      if (page > 1) {
+
+      if (limit == 0 || page == 0) {
+        limit = 20;
```

Comment:
@souravbhowmik1999 Optimise code as discussed.

Reply:
@souravbhowmik1999 You should have offset and limit and not page